src/main/java/com/fasterxml/jackson/databind/deser/BeanDeserializer.java

```diff
@@ -284,6 +286,17 @@ private final Object vanillaDeserialize(JsonParser p,
                 handleUnknownVanilla(p, ctxt, bean, propName);
             } while ((propName = p.nextFieldName()) != null);
         }
+
+        if (ctxt.hasDeserializationFeatures(DeserializationFeature.READ_NULL_OR_MISSING_CONTAINER_AS_EMPTY.getMask())) {
+            Iterator<SettableBeanProperty> itr = _beanProperties.iterator();
+            while (itr.hasNext()) {
+                SettableBeanProperty property = itr.next();
+                if (!presentNames.contains(property._propName.getSimpleName())) {
+                    property.set(bean, property._valueDeserializer.getEmptyValue(ctxt));
```

Comment:
This would basically use empty value for everything, not just collection/array/Map types. That may be beneficial in general, but makes naming of the feature mismatch.